### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by Santosh-Konappanavar**
#### Created a app.rb file with the following methods:

-  List all books.
-  List all people.
-  Create a person (teacher or student, not a plain Person).
-  Create a book.
-  Create a rental.
-  List all rentals for a given person's id.

 #### Created main.rb file with main method which:

-  Present the user with a list of options to perform.
-  Lets users choose an option.
-  If needed, ask for parameters for the option.
-  Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  puts
+end
+
+def list_rentals(rentals)
+  puts 'List of rentals:'
+  puts
+  rentals.each_with_index do |rental, index|
+    rental_info(rental, index)
+  end
+end
```
Comment (andersonlebon) [inline app.rb]: - [ ] Nice work sofar however the rental list should be filtered by person Id. kindly make sure when a user wants to display a rental they should be prompted to input the id of the person he/she want to check the rentals just like in the [video example](https://www.youtube.com/watch?v=vkkgrhD6aXQ).
image

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+def all_people(people)
+  puts 'List of people:'
+  puts
+  people.each_with_index do |person, index|
+    puts "#{index + 1}. Name: #{person.name}, ID: #{person.id}"
+    puts
+  end
+end
```
Comment (andersonlebon) [inline app.rb]: - [ ] Kindly prefix each person with his role and the age the user should know that tha person is either a student or a teacher just like in the video exaple:

your list | the video description 
----|----
image | image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Santosh-Konappanavar 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @Santosh-Konappanavar , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Good UI console design :white_check_mark: 
- [x] Good code and source file organization :white_check_mark: 
- [x]  Application works as expected :white_check_mark: 
- [x] Implemented all the changes from the previous reviewer :white_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**